### PR TITLE
Enable gen-doc on windows CI

### PR DESCRIPTION
**Pull request by shahasad**

### CODE REVIEW

File: tools/ci_build/build.py [hunk 1]
```diff
@@ -326,7 +326,7 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
                  "-Donnxruntime_CROSS_COMPILING=" + ("ON" if args.arm64 or args.arm else "OFF"),
                  "-Donnxruntime_BUILD_x86=" + ("ON" if args.x86 else "OFF"),
                   # nGraph and TensorRT providers currently only supports full_protobuf option.
-                 "-Donnxruntime_USE_FULL_PROTOBUF=" + ("ON" if args.use_full_protobuf or args.use_ngraph or args.use_tensorrt else "OFF"),
+                 "-Donnxruntime_USE_FULL_PROTOBUF=" + ("ON" if args.use_full_protobuf or args.use_ngraph or args.use_tensorrt or args.gen_doc else "OFF"),
```
Comment (pranavsharma) [inline tools/ci_build/build.py]: Any reason to turn on full protobuf when gen_doc is ON?

File: onnxruntime/python/tools/gen_doc.py [hunk 2]
```diff
@@ -114,10 +114,11 @@ def display_schema(schema, versions):  # type: (OpSchema, Sequence[OpSchema]) ->
     s = ''
 
     # doc
-    if schema.doc:
```
Comment (pranavsharma) [inline onnxruntime/python/tools/gen_doc.py]: Can we move this file to tools/python? The top-level python dir is meant for the python bindings and this script is unrelated to the bindings.

File: tools/ci_build/build.py (hunk 1 above)
Comment (shahasad) [reply tools/ci_build/build.py]: the doc string of the schema is stripped off when onnxruntime_USE_FULL_PROTOBUF is OFF

File: onnxruntime/python/tools/gen_doc.py (hunk 2 above)
Comment (shahasad) [reply onnxruntime/python/tools/gen_doc.py]: done

**Comment by natke** [comment]
Fixes #799
